pkg/device/device-util.go

```diff
@@ -177,13 +193,37 @@ func getAllPartsUsed(diskMetaName string, partitionName string) ([]partUsed, err
 		}
 		for _, tmp := range tmpList {
 			if tmp[len(tmp)-1] == partitionName {
-				partNum, _ := strconv.ParseInt(tmp[0], 10, 32)
-				pList = append(pList, partUsed{disk.DiskName, uint32(partNum)})
+				partUsed, err := parsePartUsed(disk.DiskName, tmp)
+				if err != nil {
+					return pList, err
+				}
+				pList = append(pList, partUsed)
 			}
 		}
 	}
 	return pList, nil
+}
 
+func parsePartUsed(diskName string, row []string) (PartUsed, error) {
+	p := PartUsed{DiskName: diskName}
+	if len(row) < 5 {
```

Comment:
Can we use a constant and define what `5` actually signifies. Like
```go
const noOfRowsInPartedOutput = 5
```

It would be better if a sample can also be provided, in the comments

Reply:
It can be 5 or 6 depending on if the partition has fs or not. e.g
```
$ parted /dev/sdc unit b print --script
....
Number  Start     End             Size            File system  Name                                  Flags
 1      1048576B  2097151B        1048576B                     HDD-2216723-1
 2      2097152B  9500469755903B  9500467658752B  ext4         5d8d56cb-e291-4dfd-81ac-fb664dd5ec75
 ```

Intention was to not break any consistency or refactor throughout this file (as it's using as col index numbers at every other place). Maybe we can take it up during refactoring or merging this to lvm localpv plugin. In future, we should consider to use machine parseable output of parted command (see flag -m, --machine flag) for making parsing logic easy & less error prone.

Reply:
I think that would be better, because just some integers popping up in the code causes confusion.